pootle/core/initdb.py

```diff
@@ -213,8 +223,7 @@ def create_template_languages():
     The 'templates' language is used to give users access to the untranslated
     template files.
     """
-    templates, created = Language.objects.get_or_create(code="templates",
-                                                        fullname=u'Templates')
+    _create_object(Language, **dict(code="templates", fullname="Templates"))
```

Comment:
Having a reference like `**criteria` looks fine to me, but creating the whole dict here is less readable. I suggest:

```python
    criteria = {
        'code': 'templates',
        'fullname': 'Templates',
    }
    _create_object(Language, **criteria)
```

The same for the other `**dict` appearances.